addons/web/static/src/legacy/js/core/translation.js

```diff
@@ -68,7 +68,7 @@ var TranslationDataBase = Class.extend(/** @lends instance.TranslationDataBase#
         url += '/' + (cacheId ? cacheId : Date.now());
         const paramsGet = {};
         if (modules) {
-            paramsGet.modules = modules.join(',');
+            paramsGet.mods = modules.join(',');
```

Comment:
this code is the legacy code. probably need to do the same thing in the new localization code (somewhere in `static/src/core/l10n/localization_service.js`)